src/util/each.js

```diff
@@ -18,7 +18,7 @@ export function eachNodeOrFragmentNodes(node, func) {
     const chs = node.childNodes;
     const chsLen = chs.length;
     for (let a = 0; a < chsLen; a++) {
-      func(chs[a], a);
+      func(node.cloneNode(true).childNodes[a], a);
```

Comment:
Why clone? If you append a document fragment, it's child nodes are removed from the fragment when they're transferred to the real node.